docs/sources/tempo/setup/operator/monitor.md

```diff
@@ -87,3 +92,21 @@ spec:
       sampling_fraction: "1.0"
       jaeger_agent_endpoint: localhost:6831
 ```
+
+
+## Monitor the operator
+
+The Tempo Operator can expose upgrade and other operational metrics about the operator itself, and can create alerts based on these metrics.
+The configuration options to enable the creation of `ServiceMonitor` (for scraping metrics) and `PrometheusRule` (for creating alerts) objects is in the ConfigMap `tempo-operator-manager-config` in the same namespace as the operator:
```

Comment:
This second sentence is a little confusing. Do you mean that someone has to add the example config map that you have below or that they need to set createServiceMonitors and createPrometheusRules to true in their ConfigMap?

Reply:
Is this correct? 
```suggestion
To enable metrics, you need to create the `ServiceMonitor` (for scraping metrics) and `PrometheusRule` (for creating alerts) objects. These objects are configured in the ConfigMap `tempo-operator-manager-config` in the same namespace as the operator. 

To configure these objects, set `createServiceMonitors` and `crreatePrometheusRules` to `true`. 
```

Reply:
This config map already exists, it's part of the installation of the operator. This configmap contains various settings for the operator.

When the `createServiceMonitors` setting is enabled, the operator creates this `ServiceMonitor` object in the cluster, and another operator in the cluster (for example Prometheus Operator) reads this object and configures monitoring. The user does not create this `ServiceMonitor` object, this part is handled by the operator, if the `createServiceMonitors` setting is enabled.

I've rephrased this in my last commit, I hope this clears things up? If not, please let me know :)

Reply:
Looks good! Thank you.